tests/test_redact_sample.py

```diff
@@ -45,13 +45,43 @@ def test_redact_random_htc(self):
                            'PRINT_BATCH': '2'}
 
         # When
-        redacted_sample_row = redact_sample._redact_sample_row(sample_dict_row)
+        redacted_sample_row = redact_sample._redact_sample_row(sample_dict_row, redact_htc_only=False)
 
         # Then
         self.assertTrue(1 <= int(redacted_sample_row['HTC_WILLINGNESS']) <= 5)
 
         self.assertTrue(1 <= int(redacted_sample_row['HTC_DIGITAL']) <= 5)
 
+    def test_redact_htc_only(self):
+        # Given
+        sample_dict_row = {'UPRN': '10008677190', 'ESTAB_UPRN': '10008677194', 'ADDRESS_TYPE': 'HH',
+                           'ESTAB_TYPE': 'HOUSEHOLD', 'ADDRESS_LEVEL': 'U', 'ABP_CODE': 'RD06', 'ORGANISATION_NAME': '',
+                           'ADDRESS_LINE1': 'Flat 56 Francombe House',
+                           'ADDRESS_LINE2': 'Commercial Road', 'ADDRESS_LINE3': '', 'TOWN_NAME': 'Windleybury',
+                           'POSTCODE': 'XX1 0XX', 'LATITUDE': '51.4463421', 'LONGITUDE': '-2.5924477',
+                           'OA': 'E00073438', 'LSOA': 'E01014540', 'MSOA': 'E02003043', 'LAD': 'E06000023',
+                           'REGION': 'E12000009', 'HTC_WILLINGNESS': '1',
+                           'HTC_DIGITAL': '5', 'FIELDCOORDINATOR_ID': '1', 'FIELDOFFICER_ID': '2',
+                           'TREATMENT_CODE': 'HH_LF3R2E', 'CE_EXPECTED_CAPACITY': '3', 'CE_SECURE': '0',
+                           'PRINT_BATCH': '2'}
+
+        # When
+        redacted_sample_row = redact_sample._redact_sample_row(sample_dict_row, redact_htc_only=True)
+
+        # Then
+        self.assertTrue(1 <= int(redacted_sample_row['HTC_WILLINGNESS']) <= 5)
+        self.assertTrue(1 <= int(redacted_sample_row['HTC_DIGITAL']) <= 5)
```

Comment:
This would still pass even if it wasn't altering the values.

Reply:
Ah I see it's generating random HTC values, I thought we would be setting them all to a constant. I see that makes it awkward to prove with a test which isn't ideal.

Reply:
Copied from an existing test. My bad, I should have checked. There's not really a way of testing this unless we set HTCs to a constant. We could check for number clashes but that's probably going to slow it down, and would be yucky

Reply:
Is there any reason that it can't be a constant?